helm/tests/gateway-configmap_test.yaml

```
@@ -318,3 +318,19 @@ tests:
       - matchRegex:
           path: data.[gravitee.yml]
           pattern: "[ ]{12}- /path/to/default/keystore3.pem"
+
+
+  - it: should set consent section
+    set:
+      gateway.consent.ip: true
+      gateway.consent.user-agent: false
+    asserts:
+      - hasDocuments:
+          count: 1
+      - isKind:
+          of: ConfigMap
+          path: data.[gravitee.yml]
+          pattern: |
+            consent:
+              ip: true
+              user-agent: false
```

```suggestion
      - matchRegex:
          path: data.[gravitee.yml]
          pattern: |
            consent:
              ip: true
              user-agent: false
```
Sorry, misclic when CC and GH interface issue... this should be better 😅